app/models/post.rb

```diff
@@ -1,2 +1,4 @@
 class Post < ApplicationRecord
+  validates :title, presence: true , length:{maximum: 80}
```

Comment:
please add some spacing when you write code like this `presence: true, length: { maximum: 80 }`